env_setup/env_setup.yml

```diff
@@ -9,6 +9,11 @@
   tasks:
     - include_tasks: set_default_variables.yml
 
+    - name: "Set fact of the default path of test case file"
```

Comment:
This can also be put in `set_default_variables.yml`.

Reply:
Yes, you are right. Now it is updated.